cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -37,6 +38,9 @@ const (
 
 // CreateLocalEtcdStaticPodManifestFile will write local etcd static pod manifest file.
 func CreateLocalEtcdStaticPodManifestFile(manifestDir string, cfg *kubeadmapi.InitConfiguration) error {
+	if cfg.ClusterConfiguration.Etcd.External != nil {
+		return errors.New("etcd static pod manifest cannot be generated for cluster using external etcd")
```

Comment:
Small nit
Please use fmt.Errorf, it is the current standard in kubeadm code base

Reply:
Updated